### PR TITLE
models and routes

### PR DESCRIPTION
**Pull request by rall**
This adds models for user and appointment, which behave like activerecord classes thanks to the 'Her' gem

### CODE REVIEW

File: config/initializers/her.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Her::API.setup url: 'http://api:4000' do |config|
+  config.use Faraday::Request::UrlEncoded
+  config.use Her::Middleware::DefaultParseJSON
+  config.use Faraday::Adapter::NetHttp
+end
```
Comment (waysidekoi) [inline config/initializers/her.rb]: What's going on here?

File: app/controllers/appointments_controller.rb [hunk 2]
```diff
@@ -32,7 +33,7 @@ def create
 
   # PATCH/PUT /appointments/1
   def update
-    if @appointment.update(appointment_params)
+    if @appointment.assign_attributes(appointment_params)
```
Comment (waysidekoi) [inline app/controllers/appointments_controller.rb]: why is this #assign_attributes?

**Comment by rall** [comment]
here's the http orm I'm using: https://github.com/remiprev/her (to answer your questions @waysidekoi)

**Comment by waysidekoi** [comment]
👍

**Pull request merged**